### ISSUE TITLE
Create Interactive Map of Per Facility Metrics

### ISSUE THREAD

**Issue by leggitta** [Data4Democracy/VA-open-data#19]
Each VA facility in the country has performance and spending data. Veterans and the public should be able to easily assess how their nearby VA facilities are performing.

- [ ] Gather available datasets
- [ ] Group into relevant categories
- [ ] Create interactive map showing facility data compared to other facilities around the country
- [ ] Develop intuitive performance indices for public consumption

**Comment by erevear** [comment]
@leggitta going to get started on this! Is https://www.data.va.gov/search/type/dataset the best place to get started gathering these data sets?

**Comment by leggitta** [comment]
@erevear yup! I did some work in python to read locations, but feel free to use any language you're comfortable with!
https://github.com/Data4Democracy/VA-open-data/blob/master/python/populate_database.py
